### PR TITLE
Added language in codeblocks

### PR DESCRIPTION
**Pull request by ktsrivastava29**
I'm adding languages to codeblocks in md files. There are a few code blocks that don't have a language for their lexer, and Pygments isn't always smart enough to figure out what language is intended.
The language should be stated as follows: 
````
```language

code

```
````
fix: https://github.com/plone/volto/issues/3090
also see: https://github.com/plone/volto/pull/3155

### CODE REVIEW

File: docs/source/recipes/creating-project.md
Comment (stevepiercy) [inline docs/source/recipes/creating-project.md:13]: For this numbered list, the syntax is a little tricky. Any content in a list must be indented. I also prefer to indent everything 4 spaces and have it aligned, including the content after a number or bullet. For example:

````
1.  Open a terminal and execute:

    ```shell
    npm install -g yo @plone/generator-volto
    yo @plone/volto
    ```
````

...will render nicely with the code-block within the context of item 1. Please apply the same to this list, and use `make docs-livehtml` to watch the changes as you edit them in real time.

File: docs/source/recipes/creating-project.md
Comment (stevepiercy) [inline docs/source/recipes/creating-project.md:39]:
````suggestion
```shell
````

File: docs/source/recipes/creating-project.md
Comment (stevepiercy) [inline docs/source/recipes/creating-project.md:40]: Purge all prompts.

```suggestion
yarn start
```

File: docs/source/recipes/i18n.md
Comment (stevepiercy) [inline docs/source/recipes/i18n.md:106]: See above comment about indentation of list items.

File: docs/source/recipes/i18n.md
Comment (stevepiercy) [inline docs/source/recipes/i18n.md:146]:
````suggestion
```html
````

File: docs/source/recipes/i18n.md
Comment (stevepiercy) [inline docs/source/recipes/i18n.md:137]: I'm not sure, but I think this is `jsx`.

````suggestion
```jsx
````

File: docs/source/recipes/i18n.md
Comment (stevepiercy) [inline docs/source/recipes/i18n.md:163]: This is output, not a shell command. For output, we use `console`.

````suggestion
```console
````

File: docs/source/recipes/i18n.md
Comment (stevepiercy) [inline docs/source/recipes/i18n.md:223]: See above comment about indenting list items with code-blocks.

File: docs/source/recipes/i18n.md
Comment (stevepiercy) [inline docs/source/recipes/i18n.md:270]: This one needs a language. I am not sure which. Is it Python?

File: docs/source/recipes/i18n.md
Comment (stevepiercy) [inline docs/source/recipes/i18n.md:276]:
````suggestion
```js
````

File: docs/source/recipes/lazyload.md
Comment (stevepiercy) [inline docs/source/recipes/lazyload.md:37]: We want to replace all instances of the bash language with shell so that we are shell-neutral.

````suggestion
```shell
````

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Please see individual comments.

This will also need a review from a maintainer.

File: docs/source/recipes/creating-project.md
Comment (stevepiercy) [reply docs/source/recipes/creating-project.md:13]: The code-block still does not indent properly. See the build preview:

https://deploy-preview-3165--volto.netlify.app/recipes/creating-project.html

Compare that code block to the code-blocks in https://deploy-preview-3165--volto.netlify.app/getting-started/install.html#install-nvm-nodejs-version-manager.

You can also look at the source of that file if you need further guidance.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by stevepiercy** [comment]
You will also need to resolve merge conflicts with `master` again.

**Comment by ktsrivastava29** [comment]
@stevepiercy I've now fixed the indent and have also resolved the conflicts

**Comment by stevepiercy** [comment]
@ktsrivastava29 please check the preview build and note the list is now broken.

https://deploy-preview-3165--volto.netlify.app/recipes/creating-project.html

Please build the docs locally, and inspect a preview locally, before pushing more commits.

**Comment by ktsrivastava29** [comment]
@stevepiercy I've fixed that, please let me know how can I make it better

**Comment by stevepiercy** [comment]
@ktsrivastava29 again, please check the preview. And not just for this one file, but for all files you have touched.

https://deploy-preview-3165--volto.netlify.app/recipes/creating-project.html

Before pushing any more commits to this pull request, and asking for another review, first build the docs locally and preview them to ensure that they render correctly.

**Comment by ktsrivastava29** [comment]
@stevepiercy I sincerely apologize for bothering you again; actually, I went through each file I edited and fixed the indent where necessary; locally, everything was great, but when I pushed, the test failed; could you kindly assist me in resolving this error?

**Comment by stevepiercy** [comment]
@ktsrivastava29 the failure can be resolved by rebasing your branch onto `master`, then force pushing it. We recently added the linkcheck requirement in GitHub Actions, but it is not in your branch's `Makefile`.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
When you build the docs locally, please check warnings and visually inspect the build before pushing commits and asking for my review. I have to assume you are not doing that, based on what I see in this review round. This PR still needs work. It is not OK to repeatedly say that you have addressed suggestions, but have actually not addressed them.

**Review by nileshgulia1 — COMMENTED** [review]
@ktsrivastava29 There's a simple command `make docs-livehtml` https://github.com/plone/volto/blob/master/Makefile#L127 which will save you a lot of time while developing on this PR. Please use it before pushing further commits.

**Comment by ktsrivastava29** [comment]
@stevepiercy I've fixed all and double-checked everything; I apologise for taking up so much of the time. Actually previously, I was receiving some HTTPS errors, which took me a day to resolve, and then I got an error due to the improper directory name; after resolving all of these, I was able to go through the suggested changes and then modify it.

File: docs/source/recipes/creating-project.md
Comment (stevepiercy) [inline docs/source/recipes/creating-project.md:38]: Let's keep the sentence together on one line.

```suggestion
4.  The project is ready to be started, `@plone/generator-volto` already has run the dependencies installations for you.
```

File: docs/source/recipes/creating-project.md
Comment (stevepiercy) [inline docs/source/recipes/creating-project.md:39]:
```suggestion
```

File: docs/source/recipes/environment-variables.md
Comment (stevepiercy) [reply docs/source/recipes/environment-variables.md:27]: Please also wrap lines 12 and 17 above as shell commands.

File: lib64
Comment (stevepiercy) [inline lib64:1]: This file should not be committed to the repository. Please remove.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
We're almost there! All the lists look good now. There is a file that should not be included in this PR, and a couple of other minor changes, a few which were overlooked.

**Comment by ktsrivastava29** [comment]
@stevepiercy I've made the changes you recommended; please take a look now.

File: .gitignore
Comment (stevepiercy) [inline .gitignore:12]: This is Python related, not Node. Please move this line under the #Python section, following its sorting and leading slash patterns.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
One more change, please.

**Comment by ktsrivastava29** [comment]
@stevepiercy Oh, I'm sorry, I wrongly placed it; I've fixed it now.

**Comment by ktsrivastava29** [comment]
@stevepiercy Previously, only one test failed; now, two have failed. I haven't made any changes since the last push, except for the minor change in the gitignore file that you suggested.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
The [failing docs build](https://github.com/plone/volto/runs/5708574282?check_suite_focus=true) will be fixed in https://github.com/plone/volto/pull/3196. We can ignore that failure.

For the [failing core sandbox tests build](https://github.com/plone/volto/runs/5708574126?check_suite_focus=true#step:8:738), I am not sure. @tiberiuichim or @sneridagh, have you got any idea? I restarted that failure to see if it was just a temporary connection issue. Let's see if it fails again. I restarted that failed build for my PR, and it just now passed.

**Comment by stevepiercy** [comment]
We just merged https://github.com/plone/volto/pull/3196, so the docs build should now pass, I think.

**Comment by stevepiercy** [comment]
meh, nm, docs build still breaks because this PR has not switched to use plain old `docs-linkcheck`. The docs build failure may be ignored.

**Comment by nileshgulia1** [comment]
@ktsrivastava29 You can try rebasing your PR with master.

File: lib64
Comment (stevepiercy) [reply lib64:1]: This file still exists in your PR. Please remove it from the PR.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]
Excellent. Thank you!

Would you please also either merge `master` into your branch or rebase your branch on `master` and force push? Then that should get the failing docs build to pass, and should allow you to merge your own pull request.

**Comment by stevepiercy** [comment]
Oops, actually, it looks like the docs test now passes. Nevermind about rebasing or merging. Let's wait for all green, then you can merge.

**Comment by ktsrivastava29** [comment]
@stevepiercy @nileshgulia1 Thank you; this is my third PR in this organisation, and I'm overjoyed as after 10 days it finally got approved. I'm learning a lot from your experiences. I'm eager to learn and contribute as much as I can.

**Comment by stevepiercy** [comment]
@ktsrivastava29 please do the honor of merging your PR.

**Comment by ktsrivastava29** [comment]
@stevepiercy Sorry, you will have to do it because the base branch restricts merging to me, perhaps only the core team is authorized to merge into master.

**Pull request merged**
